### ISSUE TITLE
While navigating with tab key focus is not going to "Failure description" via keyboard

### ISSUE THREAD

**Issue by DaveTryon** [microsoft/accessibility-insights-web#7218]
Copied from https://mseng.visualstudio.com/DefaultCollection/1ES/_workitems/edit/2129324

## Environment Details:
Application Name: Accessibility Insights for Web
Operating System: Windows 11 Enterprise Insider Preview(23H2)
OS Build: 26010.1000
Edge Version: Version 121.0.2256.2 (Official build) stable app, dev channel (64-bit)

## Repro Steps:
1. Hit the URL https://accessibilityinsights.io, install and add application to edge/ chrome
2. Tab till Accessibility Insights for web icon and press enter
3. Tab till ok button in Telemetry and press enter
4. Tab till Fastpass dropdown and select Assessment link and press enter
5. Select the keyboard test in the navigation pane and press enter
6. Select any requirement under this test
7. Tab till Fail radio button and select it
8. Tab to failure Description

Observe While navigating with tab key focus is not going to "Failure description" via keyboard

### Actual:
 While navigating with tab key focus is not going to "Failure description" via keyboard focus is going to direct no keyboard trap button.

### Expected:
 While navigating with tab key focus should go to "Failure description" and then it should go to no keyboard trap button via keyboard

### User Impact:
Keyboard users will face difficulty when focus is not going to "Failure description"
```[tasklist]
### Tasks
```

**Comment by DaveTryon** [comment]
We've confirmed this behavior. The new instance doesn't immediately get added to the tab order. Interestingly, if you tab past the end of the page and back around, it is then in the tab order.

**Comment by v-sharmachir** [comment]
We've confirmed this behavior is not available in newer version of Accessibility Insights for Web, hence closing it out.
